### PR TITLE
Update xml file

### PR DESCRIPTION
**Pull request by yennguyen196**

### CODE REVIEW

**Comment by yennguyen196** [comment]
ready

File: app/src/main/res/layout/activity_play_track.xml [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+        android:layout_height="@dimen/dp_50"
+        android:id="@+id/linear_toolbar"
+        android:orientation="horizontal"
+        app:layout_constraintTop_toTopOf="parent">
+
+        <ImageView
+            android:id="@+id/image_back"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:padding="10dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_play_track.xml]: hardcode

File: app/src/main/res/layout/activity_play_track.xml [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+
+    <ImageView
+        android:id="@+id/image_favorite"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="@dimen/dp_8"
+        android:layout_marginLeft="@dimen/dp_8"
+        android:layout_marginRight="@dimen/dp_8"
+        android:layout_marginStart="@dimen/dp_8"
+        android:layout_marginTop="16dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_play_track.xml]: hardcode

File: app/src/main/res/layout/activity_play_track.xml [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+    <ImageButton
+        android:id="@+id/button_state_change"
+        android:layout_width="@dimen/dp_45"
+        android:layout_height="@dimen/dp_45"
+        android:layout_marginBottom="@dimen/dp_8"
+        android:layout_marginEnd="@dimen/dp_8"
+        android:layout_marginLeft="@dimen/dp_8"
+        android:layout_marginRight="@dimen/dp_8"
+        android:layout_marginStart="@dimen/dp_8"
+        android:layout_marginTop="8dp"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_play_track.xml]: hardcode.

File: app/src/main/res/layout/activity_play_track.xml [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+        android:layout_marginEnd="@dimen/dp_40"
+        android:layout_marginRight="@dimen/dp_40"
+        android:layout_marginTop="@dimen/dp_20"
+        android:background="@drawable/ic_skip_next_black_24dp"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/image_loop"
+        app:layout_constraintTop_toBottomOf="@+id/seekBar" />
+
+    <SeekBar
+        android:id="@+id/seekBar"
```
Comment (tuannguyen38) [inline app/src/main/res/layout/activity_play_track.xml]: convention 3.2

File: app/src/main/res/layout/item_track.xml [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+        android:layout_marginTop="@dimen/dp_8"
+        android:background="@drawable/itunes"
+        android:scaleType="centerCrop"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toStartOf="@+id/guideline_three"
+        app:layout_constraintHorizontal_bias="0.0"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintVertical_bias="0.0"
+
```
Comment (tuannguyen38) [inline app/src/main/res/layout/item_track.xml]: remove bl

**Pull request merged**